web/modules/custom/joinup_subscription/joinup_subscription.post_update.php

```diff
@@ -29,6 +29,11 @@ function joinup_subscription_post_update_subscribe_to_community_content(&$sandbo
     ->execute();
 
   foreach (OgMembership::loadMultiple($mids) as $membership) {
+    // Avoid breaking the update path for leftover memberships.
+    if (empty($membership->getGroup())) {
+      trigger_error(sprintf("Group with id %s is missing.", $membership->getGroupId()), E_USER_WARNING);
+      continue;
+    }
```

Comment:
I don't understand this, isn't it allowed to delete groups in the site? I think this is by design, we are not cleaning up reverse references on purpose. Why are we throwing warnings for this?

Also, since this fixes the update path, shouldn't this be against master?

Reply:
Maybe it is better to create a PR against master.
As for the warning, it is not to declare something as miss behaving. It is merely to capture the exceptions in the output during the upgrade so that we can keep track of it.

We are already doing something similar to ensure the update path provides us enough output, e.g. the files missing and the ctt release description text copy.